mono/tests/test-driver

```diff
@@ -29,9 +29,10 @@ if ($test =~ /.*\|.*/) {
 	#This is a silly workaround, but all tests that use extra parameters need a larger timeout.
 	$timeout_in_minutes = 5;
 }
+$ENV{'TEST_DRIVER_TIMEOUT_SEC'} = $timeout_in_minutes * 60;
```

Comment:
I think there were talks about moving them to the managed test runner, so it's good we're prepared :)